pkgs/applications/networking/browsers/palemoon/default.nix

```diff
@@ -39,19 +39,21 @@
 # https://developer.palemoon.org/build/linux/
 assert stdenv.cc.isGNU;
 assert with lib.strings; (
-  versionAtLeast stdenv.cc.version "4.9"
-  && !hasPrefix "6" stdenv.cc.version
-  && versionOlder stdenv.cc.version "11"
+  versionAtLeast stdenv.cc.version "7.1"
+  && versionOlder stdenv.cc.version "12"
```

Comment:
Based on updated GCC compat matrix.

![Bildschirmfoto von 2022-05-10 22-42-24](https://user-images.githubusercontent.com/23431373/167718375-29afdf7c-24d9-47da-b990-ff7162f776a3.png)

11.x (our current default) is marked experimental, but deviating from the normal stdenv may or may not have been a cause for broken GL support on my end in the past… I didn't have the hardware to test at the time, maybe if I have more time some day I'll try to test this again.